src/webui/btjson.cpp

```diff
@@ -800,6 +801,7 @@ QVariantMap toMap(BitTorrent::TorrentHandle *const torrent)
     ret[KEY_TORRENT_NUM_INCOMPLETE] = torrent->totalLeechersCount();
     const qreal ratio = torrent->realRatio();
     ret[KEY_TORRENT_RATIO] = (ratio > BitTorrent::TorrentHandle::MAX_RATIO) ? -1 : ratio;
+    ret[KEY_TORRENT_STATUS] = torrent->state().toString();
```

Comment:
State and status use the same initial value, however the end result is different. I used different variables for the sake of clarity but they can use the same variable is desired.